src/api/DataApi.test.js

```diff
@@ -168,6 +169,35 @@ function createOrMergeEntityData() {
   });
 }
 
+function deleteAllEntitiesFromEntitySet() {
+
+  describe('deleteAllEntitiesFromEntitySet()', () => {
+
+    const apiToTest = DataApi.deleteAllEntitiesFromEntitySet;
+    const mockEntitySetId = genRandomUUID();
+
+    const validParams = [mockEntitySetId, 'Soft'];
+    const invalidParams = [INVALID_PARAMS_SS, INVALID_PARAMS];
```

Comment:
Change to `[INVALID_PARAMS_SS, INVALID_PARAMS_SS]`. The `SS` stands for "special string", which basically means enum. Since `Soft` is essentially an enum value, it must be treated as a special string.